content/en/apps/reference/api.md

```diff
@@ -1232,3 +1233,44 @@ Content-Type: application/json
   }
 }
 ```
+
+
+## Replication Limit
+
+*Added in 3.11.0*
+Returns the quantity of documents that were replicated by each user.
+Accepts filtering by user name, when not provided, it returns all users.
+Supports GET.
+Only allowed for users with "_admin" role.
+
+### GET api/v1/log-replication-limit
```

Comment:
Now that I see this written down I think we should change the endpoint URL and not use the word "log". The log is how it's stored, and not relevant to the people who call this URL.

What do you think about `api/v1/users-doc-count` ?

Reply:
Fixed.

Reply:
This line also needs to be updated...

```suggestion
### GET api/v1/users-doc-count
```

Reply:
Ups fixed.